src/ray/util/event.h

```diff
@@ -154,8 +164,13 @@ class RayEventContext final {
   std::string source_hostname_ = boost::asio::ip::host_name();
   int32_t source_pid_ = getpid();
   std::unordered_map<std::string, std::string> custom_fields_;
+  bool initialized_ = false;
 
   static thread_local std::unique_ptr<RayEventContext> context_;
+
+  static std::unique_ptr<RayEventContext> first_instance_;
```

Comment:
maybe rename it to `global_context_`. and let's add comments on what this is used for.

Reply:
OK